kessler/kessler_update.meta

```diff
@@ -23,22 +23,22 @@
   name = kessler_update_timestep_init
   type = scheme
 [ temp ]
-  standard_name = temperature
+  standard_name = air_temperature
   state_variable = true
   type = real
   kind = kind_phys
   units = K
   dimensions = (horizontal_dimension, vertical_layer_dimension)
   intent = in
 [ temp_prev ]
-  standard_name = temperature_from_previous_timestep
+  standard_name = air_temperature_on_previous_timestep
   units = K
   dimensions = (horizontal_dimension, vertical_layer_dimension)
   type = real
   kind = kind_phys
   intent = out
 [ ttend_t ]
-  standard_name = total_tendency_of_temperature
+  standard_name = total_tendency_of_air_temperature
```

Comment:
This needs to be `tendency_of_air_temperature` (or CAMDEN needs to be `total_tendency_of_air_temperature`).  Otherwise it won't match the equivalent field in CAMDEN and thus won't be linked-up properly by the CCPP.